packages/client/hmi-client/src/components/data-explorer/search-item.vue

```diff
@@ -0,0 +1,87 @@
+<script setup lang="ts">
+import { XDDArticle } from '@/types/XDD';
+import { isXDDArticle } from '@/utils/data-util';
+import IconAdd24 from '@carbon/icons-vue/es/add/24';
+import IconCheckmark24 from '@carbon/icons-vue/es/checkmark/24';
+import { ResultType } from '@/types/common';
+
+const props = defineProps<{
+	d: XDDArticle;
+	previewedArticle: XDDArticle | null;
+	selectedSearchItems: ResultType[];
```

Comment:
1. I would recommend avoid using single letter variables unless you are in a short closure/lambda situation, it makes things difficult to read, and it gets confused with other closures/lambdas where we tend to use this convention.

2. If the only purpose of previewedArticle is to check whether the item is active or not, you may want to consider bubbling it up. so instead of
```
{ XDDArticle, XDDArticle, ResultType }
```
can consider
```
{ XDDArticle, boolean, ResultType }
```
or something